tracing-futures/src/lib.rs

```diff
@@ -193,21 +193,21 @@ pub trait Instrument: Sized {
 }
 
 /// Extension trait allowing futures, streams, and sinks to be instrumented with
-/// a `tracing` [`Subscriber`].
+/// a `tracing` [`Collector`].
 ///
-/// [`Subscriber`]: https://docs.rs/tracing/latest/tracing/subscriber/trait.Subscriber.html
+/// [`Collector`]: https://docs.rs/tracing/latest/tracing/subscriber/trait.Subscriber.html
 #[cfg(feature = "std")]
 #[cfg_attr(docsrs, doc(cfg(feature = "std")))]
 pub trait WithSubscriber: Sized {
```

Comment:
this should probably be renamed to `WithCollector`

Reply:
done